src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -443,7 +443,7 @@ def _make_list_value(value: XMLValue, iri_lookup: dict[str, str]) -> dict[str, A
 def _transform_text_prop(
     prop: XMLProperty, permissions_lookup: dict[str, Permissions], iri_resolver: IriResolver
 ) -> SerialiseProperty:
-    values: list[SerialiseValue] = []
+    values = []
```

Comment:
This is another case, here it is an advantage to have a type hint. By default, `[]` is treated as `list[Any]`, so it is better to type it as `values: list[SerialiseValue] = []`